docs/queries.rst

```diff
@@ -266,5 +266,11 @@ Full example
         question = graphene.Field(QuestionType)
         questions = relay.ConnectionField(QuestionConnection)
 
+        def resolve_question(root, info, **kwargs):
+            return Question
```

Comment:
This is returning a class and not an instance.

Reply:
Just realized `question` and `resolve_question` aren't necessary to demonstrate `pagination` here.

Reply:
One thought, though. If you have the requisite `Question` models implemented on your machine, you might consider running up a quick query to follow the schema example here, with a brief discussion of `pageInfo` and cursors. I'm using my django implementation to test, so here's a sample query and result:

Query:
```
{
  places(first: 2, after: "YXJyYXljb25uZWN0aW9uOjEwNQ==") {
    pageInfo {
      startCursor
      endCursor
      hasNextPage
      hasPreviousPage
    }
    edges {
      cursor
      node {
        id
        name
      }
    }
  }
}
```

Result:
```
{
  "data": {
    "places": {
      "pageInfo": {
        "startCursor": "YXJyYXljb25uZWN0aW9uOjEwNg==",
        "endCursor": "YXJyYXljb25uZWN0aW9uOjEwNw==",
        "hasNextPage": true,
        "hasPreviousPage": false
      },
      "edges": [
        {
          "cursor": "YXJyYXljb25uZWN0aW9uOjEwNg==",
          "node": {
            "id": "UGxhY2VUeXBlOjEwNw==",
            "name": "Highland Park, Dallas, Texas, United States"
          }
        },
        {
          "cursor": "YXJyYXljb25uZWN0aW9uOjEwNw==",
          "node": {
            "id": "UGxhY2VUeXBlOjEwOA==",
            "name": "Grafton, Grafton Village, Ozaukee, Wisconsin, United States"
          }
        }
      ]
    }
  }
}
```

Reply:
@changeling go ahead and add this underneath!

Reply:
See if that works for you.

As a side note, it might be a good idea to consider query examples wherever the docs implement them or reference them.